apps/publications/src/assets/i18n/en.json

```diff
@@ -100,15 +98,17 @@
     "PREV": "Back",
     "FINISH": "Finish",
     "CREATE": "Create publication",
-    "RETURN": "Return",
+    "CANCEL": "Cancel",
     "SIMILAR_FOUND": "Similar publication already exists",
     "CHECK_BELOW": "Please check if your publication is listed below",
     "NOT_BELOW": "If NO, continue with Create button below.",
-    "IS_BELOW": "If YES, click on it to see details and add yourself as an author. Afterwards continue with the Return button. If desired publication is locked for changes, notify administrators about your request.",
+    "IS_BELOW": "If YES, click on it to see details and add yourself as an author. Afterwards continue with the Cancel button. If desired publication is locked for changes, notify administrators about your request.",
     "CHECK_PERFORM": "Checking for similar publications",
     "NO_SIMILAR": "Publication similarity check passed",
```

Comment:
I'd alter this text to something like `Check passed - no similar publications were found.`